File: PHP Excel Examples/CellsApiDemo/Row/hideRows.php
<?php

use Spire\Cloud\Excel\Sdk\Api\CellsApi;
use Spire\Cloud\Excel\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new CellsApi($configuration);

$name = "HideRows.xlsx";
$folder = "input";
$storage = null;
$sheetName = "Sheet3";
$startRow = 1;
$totalRows = 1;
$apiInstance->hideRows($name, $sheetName, $startRow, $totalRows, $folder, $storage);

File: Python Excel Examples/CellsApiDemo/row/hideRows.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration
from spirecloudexcel.api.cells_api import CellsApi

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.cells_api.CellsApi(configuration)
name = "HideRows_1.xlsx"
sheetName = "Sheet4"
startRow = 4
total_rows = 4
storage = ""
folder = "/Cells/Row/"
api.hide_rows(name, sheet_name=sheetName, startrow=startRow, total_rows=total_rows,
                   folder=folder,
                   storage=storage)